STANDARDS-3  Initial set of terms, definitions, and a dummy icon

nedhorning opened the pull request:
Created an initial set of terms, definitions, and a dummy icon for review and testing.

beccaRND commented:
@nedhorning looking at the commits sidebar list I'm seeing - symbols and some blocks are deleted. Does this mean anything significant or just GitHub being Github?

nedhorning commented:
@beccaRND I expect this is GitHub being GitHub. It's a little confusing since there are so many changes although most are either a file being deleted (red -) or added (green +). For edited files, deleted text is highlighted in red, and added text is highlighted in green. Does that help?

aaronc commented:
One issue here is that the deploy preview isn't working so we can't see the live pages. I will try to fix that

beccaRND commented on README.md:
```diff
@@ -4,13 +4,19 @@ This repository contains data standards for Regen Network.
+Each taxonomy card type can have hierarchical relationships, which will be useful when selecting, searching, and grouping terms on the web page. Below is an example. For example, if the top level of the hierarchy is “Marine,” and sub-categories under that are "Coastal", "Coral Reef", and "Deep Water". The terms for each of those levels needs a definition.
+
+In the above example, the Marine.md file would go into the “environment-type” directory (/src/content/environment-type/Marine.md) and the CoralReef.md file would go in the “Marine” directory (/src/content/environment-type/Marine/CoralReef.md)
 
```
So in this model you could have the same sub directory items under different directory items, such as coastal for marine and coastal for wetlands. that doesn't present a problem though because it's organized under a different directory item, is that correct?

nedhorning replied:
The situation you describe is possible but my take is that it should be discouraged/rejected if someone proposed that. Unless, of course, we want coastal to show up under marine and wetlands. At this point, I'd say that's for the approvers to decide using "due diligence." I expect these sorts of questions will pop up from time to time.

beccaRND commented on src/content/activity/Conservation.md:
```diff
@@ -0,0 +1,6 @@
+image: ./dummy.svg
+---
+
+The sustainable use and management of natural resources, including wildlife, water, air, and earth deposits, to ensure their availability to present and future generations.
```
could conservation include an area or resource that is not used or actively managed, more simply preserved or protected? if that is the case then it could be different from sustainable use and management.

S4mmyb replied:
Wondering the same. @nedhorning maybe it would make more sense to define it as, 

The protection, preservation, management, or restoration of natural environments and the ecological 
 communities that inhabit them. Conservation is generally held to include the sustainable use and management of natural resources including wildlife, water, air, and earth deposits, to ensure their availability to present and future generations.

nedhorning replied:
I followed Sam's definition but removed "restoration" since that's covered in EcosystemRestoration.

beccaRND commented on src/content/impact/ImprovedBiodiversity.md:
```diff
@@ -0,0 +1,7 @@
+image: ./dummy.svg
+---
+
+Protect or restore the variety of life at genetic, species and ecosystem scales, in an area, including different species of plants, animals, and other organisms.
```
the definition is correct for the md Improved Biodiversity but the title says increased carbon sequestration and storage

S4mmyb replied:
suyggested change above

nedhorning replied:
I changed this to the correct title

beccaRND commented on src/content/impact/ImprovedWaterInfiltration.md:
```diff
@@ -0,0 +1,7 @@
+image: ./dummy.svg
+---
+
+Enhance the ability of water to soak into the ground, improving groundwater recharge and reducing surface runoff. 
```
because infiltration is measured as a rate, should the definition include "enhance the rate and ability of water..."

nedhorning replied:
I added "rate" as you suggested

beccaRND reviewed (approved):
Comments left for your review; most are suggested changes. The only comment that must be addressed is the Improved biodiversity field having the wrong title.

S4mmyb commented on src/content/activity/SoilAmendments/Biochar.md:
```diff
@@ -0,0 +1,6 @@
+image: ./dummy.svg
+---
+
+Adding biochar to the soil to improve its quality and fertility.
```
but the end effect is still the same, right? @beccaRND did you have a proposed upgrade? Or should we keep it as is

nedhorning replied:
I left this as is for the reason Sam highlights, but can change it if you think it should explicitly state that biochar does not directly impact soil quality or something like that.

beccaRND replied:
in the end the effect is not the same if nutrients are not present or added. my proposed upgrade would be to say "adding biochar to the soil to improve its quality and nutrient and water holding capacity."

nedhorning replied:
I like that. I changed the text as you suggested

S4mmyb commented on src/content/activity/SoilAmendments.md:
```diff
@@ -0,0 +1,6 @@
+image: ./dummy.svg
+---
+
+Practices that modify the soil to improve its physical or chemical properties by applying specific amendments.
```
I think it's probably fine as is personally

nedhorning replied:
I added: Soil amendments should not be confused with fertilizer which is meant to supply nutrients to the soil.

S4mmyb commented on src/content/impact/AvoidedDeforestationDegradation.md:
```diff
@@ -0,0 +1,6 @@
+image: ./dummy.svg
+---
+
+Prevent the loss or degradation of forests.
```
standing forests usually refer to old groth though, correct? I think we want to support all forests

nedhorning replied:
I prefer to leave this as is. My interpretation of standing forests is forests with trees growing as opposed to harvested. I think "forest" typically  creates a mental image of an area with trees.

S4mmyb commented on src/content/impact/ImprovedBiodiversity.md:
```diff
@@ -0,0 +1,7 @@
+---
+title: Increased Carbon Sequestration and Storage
```
```suggestion
title: Improved Biodiversity
```

S4mmyb commented on src/content/impact/ImprovedWaterInfiltration.md:
```diff
@@ -0,0 +1,7 @@
+image: ./dummy.svg
+---
+
+Enhance the ability of water to soak into the ground, improving groundwater recharge and reducing surface runoff. 
```
```suggestion
Enhance the rate and ability of water to soak into the ground, improving groundwater recharge and reducing surface runoff. 
```

S4mmyb reviewed (commented):
Hey @nedhorning , overall i think this looks great! Becca had a few comments for suggested changes. Once those are corrected we can approve and merge. Nice job!

nedhorning commented on src/content/activity/SoilAmendments/EnhancedRockWeathering.md:
```diff
@@ -0,0 +1,6 @@
+image: ./dummy.svg
+---
+
+This carbon removal activity often also adds micronutrients to the soil, improving soil health.
```
I could use an example. I'm not sure what you mean by "fast-track". Do you mean relative to the carbon removal rate before it was crushed? Even after it's crushed it's still kinda slow a removing carbon. I'm trying to not go too deep into the weeds with definitions unless it's necessary to add clarity.

S4mmyb replied:
I think its fine to leave as is.

nedhorning commented on src/content/impact/ImprovedForestHealth.md:
```diff
@@ -0,0 +1,7 @@
+image: ./dummy.svg
+---
+
+Improve the condition of forests, making them more resilient to pests, diseases, and environmental stressors.
```
I added: human interventions, and other environmental stressors

nedhorning commented on src/content/impact/ReducedFertilizerAmendments.md:
```diff
@@ -0,0 +1,7 @@
+image: ./dummy.svg
+---
+
+Reduction of natural or synthetic nutrients typically applied to the soil to boost plant growth.
```
I added " and yields"

nedhorning commented on src/content/activity/Agroforestry/RiparianForestBuffers.md:
```diff
@@ -0,0 +1,6 @@
+image: ./dummy.svg
+---
+
+Strips of trees and vegetation planted along waterways with the intent to help filter pollutants, prevent soil erosion, stabilize stream banks, and enhance wildlife habitat, contributing to improved water quality and ecosystem health.
```
Yes - I mistakenly put that under Agroforestry. I moved the file up one level
